pkg/secrethub/client.go

```diff
@@ -11,6 +12,7 @@ import (
 type Client interface {
 	AccessRules() AccessRuleService
 	Accounts() AccountService
+	Auth() AuthService
```

Comment:
Is it `Auth()` or `Sessions()`?

Reply:
As said before, we could rename the client side of this to session, I can agree with that.